cmd/bots/internal/environment.go

```diff
@@ -35,107 +37,148 @@ var (
 	templateFiles embed.FS
 )
 
+type expectedMsgType byte
+
+const (
+	Html expectedMsgType = iota
```

Comment:
Use `iota+1` for first enum value. (Now `expectedMsgType == 0` and it's default zero value for integers, so empty struct which contains such field would have `expectedMsgType`).